test/e2e/manifests/mysql.yaml

```diff
@@ -267,6 +267,15 @@ subjects:
 apiVersion: v1
 data:
   config: |
+    executor:
```

Comment:
Are these file auto-generated using `make manifest` (@alexec)? If you edited these files manually they might get overwritten next time someone runs it

Reply:
@simster7 yeah, I edited them manually. Let me check if I can find a source

Reply:
@simster7 is correct. These are based on manifests/... and test/e2e/manifest/...

Reply:
@simster7 @alexec should be okay by now.